docs/dev/batch/examples.md

```diff
@@ -27,8 +27,7 @@ The following example programs showcase different applications of Flink
 from simple word counting to graph algorithms. The code samples illustrate the
```

Comment:
In my opinion, Tutorials and Examples are two different things.

* Tutorials are step-by-step guides to solve a particular task
* Examples show an API can be used to implement different tasks

Right now, there is no page about streaming examples. Adding this would be a follow up issue, IMO.

Reply:
Thanks for you explanation. Now I agree with you.